<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>score</class>
 <widget class="QDialog" name="score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;          
font-size: 20px;         
font-family: Arial;
font-weight: bold;
text-align: center;</string>
       </property>
       <property name="text">
        <string>Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scoreBtn">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 80px;                
                height: 50px;      </string>
       </property>
       <property name="text">
        <string>score</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;          
font-size: 20px;         
font-family: Arial;
font-weight: bold;
text-align: center;</string>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timeBtn">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 100px;                
                height: 50px;      </string>
       </property>
       <property name="text">
        <string>time</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>181</number>
     </property>
     <item>
      <widget class="QPushButton" name="finishBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
       </property>
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="feedbackArea">
     <property name="maximumSize">
      <size>
       <width>650</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; 
                color: black;             
                border: 2px solid white;   
                border-radius: 20px;         
                padding: 10px;            
                font-size: 14px;            
                font-family: Arial; </string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="level_lbl">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;          
font-size: 20px;         
font-family: Arial;
font-weight: bold;
text-align: center;</string>
     </property>
     <property name="text">
      <string>level</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
